### DISCUSSION TITLE
why is the SliderThumb isInvalid property deprecated?

### DISCUSSION THREAD

**Discussion by binaryartifex** [adobe/react-spectrum#6598] [Q&A]
Is there a reason why the SliderThumb isInvalid property is deprecated? There's no isInvalid alternative on the SliderTrack or the Slider container either. Its all good and well to be able to constrain the min/max client side but again, and the same goes with the switch, if we're adding these components to a form, you've taken away the ability to pass back down server side business validation that may have been attributed to those particular inputs that wasn't able to be inferred client side prior to submission, there might be dependencies in a complex form that render certain values invalid...is there an RAC composition way to still achieve this?

EDIT: so im trying to get this component to integrate nicely with react-hook-form. so there's no 'name' or 'onBlur' properties on the Slider container either. Is there a way to have a name on the parent that at least could be optionally passed as a prefix on the underlying thumbs and a parent onBlur that detects blur outside of the entire component itself?

**Comment by snowystinger** [comment]
Going to close this as a duplicate or in the umbrella of https://github.com/adobe/react-spectrum/issues/6151
